src/dotnet/APIView/APIViewWeb/wwwroot/css/site.css

```diff
@@ -66,11 +66,11 @@ body {
 }
 
 .enum {
-    color: #8700BD;
+    color: #8700BD !important;
 }
 
 .class {
-    color: #00B8B1;
+    color: #00B8B1 !important;
```

Comment:
Haha, you've found `!important`. What rule are you trying to override?

Reply:
I tried to get around using it but the definition of an anchor tag without an href inherits color from its parent, so all class and enum definitions rendered without any color. I looked at the CSS from the webpage and the definition was in a file that I don't seem to have access to, so I resorted to `!important`